node/login-system/login-system-using-node-express.md

```diff
@@ -0,0 +1,261 @@
+## Login System using Node/Express
+
+In this tutorial, I would walk you through how you can create a login system with node/express/mongodb using bcrypt to hash passwords and jwt (jsonwebtoken) for session management.
+
+> Tech Stack: Node/Express/Ejs/MongoDB/BCrypt/JWT
+
+Lets get started
+
+Create a folder on Desktop as 'login-system' and initialize the project in there.
+
+    cd Desktop
+    mkdir login-system
+    cd login-system
+    npm init
+
+Change the entry point as _server.js_ and author as _your name_
```

Comment:
It's a best practice to not use italic in professional blog posts. Maybe use bold instead.